beeai/agents/rebase_agent.py

```diff
@@ -185,12 +185,23 @@ async def commit_push_and_open_mr(state):
             state.merge_request_url = await tasks.commit_push_and_open_mr(
                 local_clone=state.local_clone,
                 files_to_commit="*.spec",
-                commit_message=f"{COMMIT_PREFIX} Update to version {state.version}",
+                commit_message=(
+                    f"{COMMIT_PREFIX} resolves {state.jira_issue}\n\n"
+                    f"Resolves: {state.jira_issue}\n\n"
+                    "This commit was created by Jotnar, a Red Hat Enterprise Linux packaging AI agent.\n\n"
+                    f"Assisted-by: Jotnar\n"
+                ),
                 fork_url=state.fork_url,
                 dist_git_branch=state.dist_git_branch,
                 update_branch=state.update_branch,
                 mr_title=f"{COMMIT_PREFIX} Update to version {state.version}",
-                mr_description="TODO",
+                mr_description=(
+                    f"This merge request was created by Jotnar, a Red Hat Enterprise Linux packaging AI agent.\n"
+                    "Carefully review the changes and make sure they are correct.\n\n"
```

Comment:
Since this repeats, maybe it could be a constant?

Reply:
yeah, we can do both